### ISSUE TITLE
Obsolete

### ISSUE THREAD

**Issue by jimlynnjulian** [pdrm83/rotten_tomatoes_scraper#7]
This program is obsolete.

C:\Users\jimly\Downloads\rotten_tomatoes_cli-master>pip3 install rotten-tomatoes-scraper
Requirement already satisfied: rotten-tomatoes-scraper in c:\users\jimly\appdata\local\programs\python\python310\lib\site-packages (1.4.0)
Requirement already satisfied: bs4 in c:\users\jimly\appdata\local\programs\python\python310\lib\site-packages (from rotten-tomatoes-scraper) (0.0.1)
Requirement already satisfied: requests in c:\users\jimly\appdata\local\programs\python\python310\lib\site-packages (from rotten-tomatoes-scraper) (2.30.0)
Requirement already satisfied: lxml in c:\users\jimly\appdata\local\programs\python\python310\lib\site-packages (from rotten-tomatoes-scraper) (4.9.2)
Requirement already satisfied: beautifulsoup4 in c:\users\jimly\appdata\local\programs\python\python310\lib\site-packages (from bs4->rotten-tomatoes-scraper) (4.12.2)
Requirement already satisfied: charset-normalizer<4,>=2 in c:\users\jimly\appdata\local\programs\python\python310\lib\site-packages (from requests->rotten-tomatoes-scraper) (2.1.1)
Requirement already satisfied: idna<4,>=2.5 in c:\users\jimly\appdata\local\programs\python\python310\lib\site-packages (from requests->rotten-tomatoes-scraper) (2.5)
Requirement already satisfied: urllib3<3,>=1.21.1 in c:\users\jimly\appdata\local\programs\python\python310\lib\site-packages (from requests->rotten-tomatoes-scraper) (1.25.11)
Requirement already satisfied: certifi>=2017.4.17 in c:\users\jimly\appdata\local\programs\python\python310\lib\site-packages (from requests->rotten-tomatoes-scraper) (2017.4.17)
Requirement already satisfied: soupsieve>1.2 in c:\users\jimly\appdata\local\programs\python\python310\lib\site-packages (from beautifulsoup4->bs4->rotten-tomatoes-scraper) (2.3.2.post1)

**Comment by pdrm83** [comment]
Yes. This package doesn't get any updates anymore. Therefore, it gets obsolete.
